### ISSUE TITLE
National Theatre

### ISSUE THREAD

**Issue by Rotajona** [cjdnrota/Rota-marketing#10]
Lisa Burger:
lisabu@hotmail.co.uk

Director @ National Theatre - think this would be a great resource for good members - loads of actors needing to pay rent etc....

I will email for a warm intro if you like.

**Comment by jenschuring** [comment]
Yes, that would be great! If you could get us in touch, then we will take it from there!
